### PR TITLE
psact update 2

### PR DESCRIPTION
**Pull request by HarrisonWAffel**
https://github.com/rancher/rancher/issues/38701

### CODE REVIEW

File: pkg/crds/dashboard/crds.go [hunk 1]
```diff
@@ -46,6 +46,13 @@ func List(cfg *rest.Config) (_ []crd.CRD, err error) {
 			c.GVK.Version = "v1"
 			return c
 		}),
+		newCRD(&v3.PodSecurityAdmissionConfigurationTemplate{}, func(c crd.CRD) crd.CRD {
+			c.Status = true
```
Comment (snasovich) [inline pkg/crds/dashboard/crds.go:50]: I don't believe we need Status subresource for this, so it should probably be `false`. Did it require `true` in your testing for some reason?

**Review by snasovich — CHANGES_REQUESTED** [review]

File: pkg/crds/dashboard/crds.go (hunk 1 above)
Comment (HarrisonWAffel) [reply pkg/crds/dashboard/crds.go]: No, I've removed it.

**Comment by HarrisonWAffel** [comment]
@jiaqiluo @kinarashah this is now ready for review, sorry for the delay

**Review by snasovich — APPROVED** [review]

**Review by jiaqiluo — CHANGES_REQUESTED** [review]
Can you also update the commit message?

File: pkg/crds/dashboard/crds.go [hunk 2]
```diff
@@ -46,6 +46,13 @@ func List(cfg *rest.Config) (_ []crd.CRD, err error) {
...
+			c.NonNamespace = true
+			c.GVK.Kind = "PodSecurityAdmissionConfigurationTemplate"
+			c.GVK.Version = "v3"
```
[the remark this answers is not in the document]
Comment (snasovich) [reply pkg/crds/dashboard/crds.go:52]: I'd say if we don't _have_ to add group and framework determines it automatically then better not to add it.

**Review by snasovich — DISMISSED** [review]

**Review by jiaqiluo — APPROVED** [review]

**Comment by HarrisonWAffel** [comment]
PR has been rebased to resolve build errors

**Comment by HarrisonWAffel** [comment]
Build is now passing

**Review by jiaqiluo — APPROVED** [review]

**Pull request merged**